tesseract_common/include/tesseract_common/class_loader.hpp

```diff
@@ -186,7 +186,7 @@ std::vector<std::string> ClassLoader::getAvailableSections(const std::string& li
     if (include_hidden)
       return false;
 
-    return (section.substr(0, 1) == ".");
+    return (section.substr(0, 1) == ".") || (section.substr(0, 1) == "_");
```

Comment:
Why was this change needed?

Reply:
Private symbols appear to start with underscores on macos